modules/booksFunction.js

```diff
@@ -0,0 +1,56 @@
+import BookStore from './books.js';
+
+const newBookStorage = new BookStore();
+
+const bookStorage = JSON.parse(localStorage.getItem('bookCollection'));
+if (bookStorage !== null) {
+  newBookStorage.store = bookStorage;
+}
+
+export function loadBooksCollection() {
```

Comment:
Please refactor this function to use arrow syntax to adhere to the project requirements🙏